README.md

```diff
@@ -404,3 +404,9 @@ Follow our [H2O Stream](https://groups.google.com/forum/#!forum/h2ostream).
     import _root_.hex.tree.gbm.GBM
     ```
   
+* Getting non-deterministic H2O Frames after the Spark Data Frame to H2O Frame conversion.
```

Comment:
I would mention there this one https://issues.apache.org/jira/browse/SPARK-17806, but just  just as related issue since  since I was still able to reproduce on Spark's 2.1.1 snapshot.